docker-compose.example.yml

```diff
@@ -1,28 +1,36 @@
-version: "3.7"
+version: "2.4"
 services:
   db:
     image: postgres:11 # pin postgres to a major version
+    # Container_name is an optional value for single-container deployments to set a friendly container name.
+    # Disable in swarm or other multi-container setups.
+    container_name: babybuddydb
     environment:
       - PGDATA=/db-data
       - POSTGRES_PASSWORD=postgres # has to correspond with DB_PASSWORD in APP
     volumes:
       - db:/db-data:rw
+    restart: unless-stopped
   app:
     image: babybuddy/babybuddy
+    # Container_name is an optional value for single-container deployments to set a friendly container name.
+    # Disable in swarm or other multi-container setups.
+    container_name: babybuddyapp
     # See README.md#configuration for other environment configuration options.
     environment:
-      - ALLOWED_HOSTS=
-      - DB_PASSWORD=postgres # has to correspond with POSTGRES_PASSWORD in DB
```

Comment:
One small thing -- this is a recent change that should be kept, please revert this line and remove the old `POSTGRES_PASSWORD` below (L25). And the new inline comment should read, "Must correspond with POSTGRES_PASSWORD in DB"

Reply:
I see, corrected, thanks!